### PR TITLE
Update EIP Bot

### PR DESCRIPTION
**Pull request by Pandapip1**
The current EIP Bot version is out of date. This PR bumps it to the most recent version.

Blocker: https://github.com/ethereum/EIP-Bot/pull/105

### CODE REVIEW

**Review by MicahZoltu — APPROVED** [review]

**Comment by Pandapip1** [comment]
Uh, this wasn't ready to merge. It wasn't even the latest version as of when this was merged 😆

**Comment by MicahZoltu** [comment]
Ah, sorry.  Once we get multiple reviewers required setup we can go back to using DRAFT as a meaningful thing.  FWIW, I probably wouldn't have approved it if you set it to the latest.  I approved because when I reviewed the changeset between the two versions it was all completely innocuous.

In the future, I'll leave it to the author to remove the DRAFT tag, rather than doing that bit myself.  Sorry!

**Pull request merged**
